rct229/rulesets/ashrae9012019/ruleset_functions/get_zone_target_baseline_system.py

```diff
@@ -192,7 +192,9 @@ def get_zone_target_baseline_system(
             zones_and_systems_b[zone_id_b] = {
                 "system_origin": SYSTEMORIGIN.G311F,
                 "expected_system_type": expected_system_type_from_table_g3_1_1_dict(
-                    get_zone_hvac_bat_dict(rmd_b, zone_id_b),
+                    list(get_zone_hvac_bat_dict(rmd_b, zone_id_b).keys())[
+                        0
+                    ],  # TODO: check if the return value always has one BAT. Otherwise, this is wrong.
```

Comment:
1. The function `get_zone_hvac_bat_dict` guaranteed provide a dict that contains the HVAC building types.
2. We should rank the one with hightest floor area instead of taking out the first one.
you can do:
```python
zone_hvac_bat_dict_b = get_zone_hvac_bat_dict(rmd_b, zone_id_b)
expected_system_type_from_table_g3_1_1_dict(
      max(zone_hvac_bat_dict_b, key=zone_hvac_bat_dict_b.keys()),
      ...
)
```

Reply:
Good point! Addressed!